.github/workflows/integration-tests.yaml

```diff
@@ -12,11 +12,47 @@ jobs:
         os: [ubuntu-latest, windows-latest]
     runs-on: ${{ matrix.os }}
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
+      - name: Checkout the current branch of mizu-client repo here.
+        uses: actions/checkout@v3
+
+
+      - name: Set up Docker Buildx
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+        uses: docker/setup-buildx-action@v2
+      # - name: Login to DockerHub (we can even login to other container registries)
+      #   if: ${{ matrix.os == 'ubuntu-latest' }}
+      #   uses: docker/login-action@v2
+      #   with:
+      #     username: ${{ secrets.DOCKERHUB_USERNAME }}
+      #     password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build and push
```

Comment:
If we're not currently pushing to DockerHub, we should choose a more accurate name for this build step.

Reply:
Ha, yes need to be a common name like "Login to container registry"
I was unsure which container registry you prefer/use. So i used dockerhub and commented for time being.

Reply:
Thanks. Leaving it commented out is a good idea, will make it easier to add a container registry later.

Reply:
updated build step name to Build